ci-operator/config/openshift/osde2e/openshift-osde2e-master.yaml

```diff
@@ -0,0 +1,25 @@
+build_root:
+  image_stream_tag:
+    cluster: https://api.ci.openshift.org
+    name: release
+    namespace: openshift
+    tag: golang-1.11
```

Comment:
Just wanted to check this is the right version of Go for you

Reply:
Yeah, this is what we use in the osde2e  Dockerfile. Thanks for checking though.